src/Microsoft.OData.Core/Metadata/ODataMetadataWriterUtils.cs

```diff
@@ -32,7 +32,11 @@ internal static XmlWriter CreateXmlWriter(Stream stream, ODataMessageWriterSetti
 
             XmlWriterSettings xmlWriterSettings = CreateXmlWriterSettings(messageWriterSettings, encoding);
 
+            #if NETSTANDARD1_1

```

Comment:
You should not add space to the `#if` preprocessor directive. Look at this file and format accordingly
https://github.com/OData/odata.net/blob/3e2d0cdbffd19bb3803516ba7841de18618a7850/src/Microsoft.OData.Client/DynamicProxyMethodGenerator.cs

Reply:
@KenitoInc Done